samples/snippets/src/test/java/com/example/spanner/admin/generated/SampleTestBaseV2.java

```diff
@@ -113,6 +113,7 @@ public static void afterClass() throws InterruptedException {
       }
     }
     for (String backupId : idGenerator.getBackupIds()) {
+      System.out.println("Trying to delete " + backupId);
```

Comment:
nit: I assume that this and the other System.out.println will be removed before merging.

Reply:
This is actually pretty useful. At the end of the test it prints if all the resources were cleaned up or not.